explorer/notebooks/ArcgisConnect.ipynb

```diff
@@ -16,52 +16,43 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 12,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Enter password: ········\n"
```

Comment:
why have this enter password thing? shouldn't it be an env variable now?

Reply:
If it's not setup it will ask for your password as a prompt, pretty neat.

Reply:
cool. could it be saved by accident or is it all safe?